fix: LEAP-443: Select only one annotation on init

robot-ci-heartex opened the pull request:
Hi @hlomzik!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/7167339170) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-frontend/pull/1640
- https://github.com/HumanSignal/dm2/pull/298

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7167339170

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7171454122

hlomzik committed:
Merge branch 'origin/develop' into 'fb-leap-443/lsf-init'

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7172896149

hlomzik committed:
Merge branch 'origin/develop' into 'fb-leap-443/lsf-init'

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7192485178

hlomzik committed:
Merge branch 'origin/develop' into 'fb-leap-443/lsf-init'

nikitabelonogov committed:
Merge remote-tracking branch 'origin/develop' into fb-leap-443/lsf-init

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7458170200

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7458170200

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7458365827

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7699356098

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7699367657

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7701471997

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7712190228

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7712200179

hlomzik committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7732120783

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7734240737

hlomzik commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7734360487)
> Unresolvable merge conflict detected, please resolve it using the git command line: label_studio/frontend/dist/lsf/js/main.js
label_studio/frontend/dist/lsf/js/main.js.map
label_studio/frontend/dist/lsf/version.json

hlomzik committed:
Merge branch 'develop' into fb-leap-443/lsf-init

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7734540515

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7734568169

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7734621712

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7734638758

hlomzik committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7734735209

hlomzik committed:
Trigger CI

jombooth committed:
relock

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7740842805

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7740887210

hlomzik committed:
Merge branch 'develop' into 'fb-leap-443/lsf-init'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7807767613

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7807767613

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7807786589

hlomzik committed:
Merge branch 'develop' into 'fb-leap-443/lsf-init'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7913722034

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7913722034

hlomzik committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7913740311

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7913775188

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7915051065

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7915086002

hlomzik committed:
Merge branch 'develop' into 'fb-leap-443/lsf-init'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7959988539

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7959988539

hlomzik committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7959988539

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7960021599

hlomzik committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7968783916

hlomzik committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7968783916

hlomzik committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7968794643

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7968817174
